apps/server/src/modules/user-import/loggable/user-migration-already-exist.loggable.ts

```diff
@@ -0,0 +1,14 @@
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+
+export class UserAlreadyExistLoggable implements Loggable {
```

Comment:
The file name is user-migration-already-exist, but the class name is UserAlreadyExist. Also I think UserAlreadyMigrated would be better.